src/server/controller/user.js

```diff
@@ -39,7 +39,7 @@ router.get('/auth/github', checkReturnTo);
 // router.get('/auth/github/callback', passport.authenticate('github', { successReturnToOrRedirect: '/', failureRedirect: '/' }));
 router.get('/auth/github/callback', passport.authenticate('github', { failureRedirect: '/' }),
     function (req, res) {
-        if(req.user && req.session.requiredScope != 'public' && ( !req.user.scope || req.user.scope.indexOf('write:repo_hook') < 0 )) {
+        if(req.user && req.session.requiredScope != 'public' && req.user.login == config.server.github.admin_user && ( !req.user.scope || req.user.scope.indexOf('write:repo_hook') < 0 )) {
```

Comment:
This statement doesn't work as optional. If I haven't set the GITHUB_ADMIN_USER, I have no chance to sign in as an admin 
!fix